examples/common/python/Makefile

```diff
@@ -24,6 +24,7 @@ all : $(WHEEL_FILE)
 
 $(WHEEL_FILE) : build_ext
 	@echo Build Distribution
+	pip3 install wheel
```

Comment:
Is this change to fix the stand-alone build?

Reply:
@Ram-srini : Yes

Reply:
I will remove the `pip3 install wheel` and modify instead the PREREQUISITES.md to install pip3 as non-root. This should fix this problem and other problems.

If the pip3 packages also need to be installed as root, let me know.